### PR TITLE
Updated Mobile Performance, Reformatted

### PR DESCRIPTION
**Pull request by DavidLu1997**
Added S6 performance, reformatted remaining performance specs, added wifi optimal performance.

Minor changes:
- Swapped upload/download of 6s plus (50mb up and 12 mb down does not make sense)
- Reworded specs slightly
- Changed formatting of speeds

### CODE REVIEW

**Comment by it33** [comment]
Thanks @DavidLu1997!

So... um... please keep technical updates and revert the formatting changes? 

"< 4 s" is not as clear as "less than 4 seconds". Please ping LI and ask her to order you [this book](http://www.amazon.com/Dont-Make-Me-Think-Usability/dp/0321344758), print or digital however you prefer. 

Please skim it and let's discuss? You've got solid skills, we need to hone them a bit, 

@esethna we need to figure out a style guide,

**Comment by DavidLu1997** [comment]
@it33 thank you :grin: 

- Reverted formatting changes
- Changed <4s to `less than 4 seconds`
- Made minor technical corrections

File: source/deployment/push.rst [hunk 1]
```diff
@@ -150,31 +150,39 @@ Performance Benchmarks for Mobile Applications
...
-iPhone Plus 6s on 4G/LTE connection (50 ms ping time, 50 Mbps upload, 8 Mbps download): 
+iPhone 6s Plus on 4G/LTE connection (50 ms ping time, 50 Mb/s download, 8 Mb/s upload): 
 
-- **Loading a new channel:** less than 4 seconds 
+- **Loading a new channel:** less than 4 seconds
 - **Returning to a channel previously viewed:** less than 1 second
-- **Switching back to app after it's recently been in the background:** less than 1 second 
-- **Switching to app and loading channel after phone has been asleep:** less than 5 seconds 
-- **Cold start of app until first page load:** 10 seconds 
+- **Switching to the app after previously opening it:** less than 1 second
```
Comment (esethna) [inline source/deployment/push.rst]: Hey @DavidLu1997, I'm not sure that this is very clear. I'd propose the original wording is probably okay?

**Comment by DavidLu1997** [comment]
Changed :)

**Comment by esethna** [comment]
Thanks for the tweaks! Merged.

**Pull request merged**